src/ImageRunModal.jsx

```diff
@@ -355,6 +395,65 @@ export class ImageRunModal extends React.Component {
                 });
     }
 
+    onCreateClicked() {
+        const createConfig = this.getCreateConfig();
+        const { owner, runImage } = this.state;
+
+        // TODO: correct?
+        let isSystem = false;
+        if (this.state.image && this.state.image.isSystem) {
+            isSystem = true;
+        }
+        if (!this.state.image && owner === systemOwner) {
+            isSystem = true;
+        }
+
+        client.imageExists(isSystem, createConfig.image).then(reply => {
+            this.createContainer(isSystem, createConfig, runImage);
+        })
+                .catch(ex => {
+                    console.info('image does not exists pull');
```

Comment:
This is rather cryptic message :) Also if it fails, would make sense to show it in alert as we show other errors?